.circleci/config.yml

```diff
@@ -61,12 +61,13 @@ jobs:
           command: OPENPBTA_SUBSET=0 ./scripts/run_in_ci.sh bash analyses/molecular-subtyping-LGAT/run_subtyping.sh
 
       - run:
-         name: Molecular Subtyping / Reclassification - EWS
-         command: ./scripts/run_in_ci.sh bash analyses/molecular-subtyping-EWS/run_reclassification.sh
+         name: Molecular Subtyping - EWS
+         command: ./scripts/run_in_ci.sh bash analyses/molecular-subtyping-EWS/run_subtyping.sh
 
-      - run:
-         name: Molecular Subtyping - Compile and incorporate pathology feedback
-         command: OPENPBTA_TESTING=1 ./scripts/run_in_ci.sh bash analyses/molecular-subtyping-pathology/run-subtyping-aggregation.sh
```

Comment:
I don't know enough about what output from here `run-subtyping-aggregation.sh`. Just wanna double check that the new output in `EWS_results.tsv` still has all the necessary components that `run-subtyping-aggregation.sh` will ultimately need?

Reply:
Yeah I think it does have all columns mainly BS IDs with matching molecular_subtype. Earlier we were "reclassifying" the samples but now as per #804  we will subtype only and so I've tried to keep the output similar to molecular-subtyping-HGG module